civictechprojects/migrations/0002_auto_20171119_2223.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.7 on 2017-11-19 22:23
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('civictechprojects', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='project',
+            name='project_issue_area',
+            field=models.CharField(default='Political Reform', max_length=200),
```

Comment:
Nope, by bad. Obviated by 9576fdcdbabf08633caa4734d04a470274bf0886

Reply:
To my understanding, migrations shouldn't be removed because they are analogous to commits in a VCS -- instead of removing old migrations/commits, new ones are merely stacked atop the old.

https://docs.djangoproject.com/en/1.11/topics/migrations/#the-commands